understanding/20/audio-description-prerecorded.html

```diff
@@ -16,12 +16,20 @@ <h2>Intent of Audio Description (Prerecorded)</h2>
       <p>The intent of this Success Criterion is to provide people who are blind or visually
          impaired access to the visual information in a synchronized media presentation. The
          audio description augments the audio portion of the presentation with the information
-         needed when the video portion is not available. During existing pauses in dialogue,
-         audio description provides information about actions, characters, scene changes, and
-         on-screen text that are important and are not described or spoken in the main sound
-         track.
+         needed when the video portion is not available. Audio description provides information about actions,
+         characters, scene changes, and on-screen text that are important and are not described
+         or spoken in the main sound track. Audio description is added during existing
+         pauses in the dialogue.
       </p>
       
+      <div class="note">
+         <p>This success criterion applies even in cases where the original audio does not
+            have sufficient pauses, and where no portion of the original audio can be omitted in
```

Comment:
```suggestion
            have sufficient pauses in the dialog to provide any audio descriptions, then the content fails this success criterion. The information can still be made available to users either through other time-based media requirements. As noted in the [Time-based media guideline](https://www.w3.org/WAI/WCAG22/Understanding/time-based-media), "Sometimes, there is so much dialogue that audio description cannot fit into existing pauses in the dialogue. The option at Level A to provide an alternative for time-based media instead of audio description for synchronized media would allow access to all of the information in the synchronized media. This option also allows access to the visual information in non-visual form when audio description is not provided for some other reason."
```

Reply:
I was a little surprised to find the text I've cited at the guideline level, and I'm not entirely satisfied with what it says; however, given it is original 2.0 text and is still in place, I thought our guidance should align with it as much as possible-- or we should get consensus to change it.
IMO, in the situation quoted here, the user would meet AA even if they did not provide the additional content in a media alternative at A. They have fulfilled the wording of the AA requirement which really does seem to support the read that once the available space is used up, it's been met.

Reply:
I'm a bit stumped about the first change (spread over two lines) that says

```
However, the information can still be made available to users through other time-based media requirements at A and AAA levels.
```

What is this getting at? We're in the understanding for AAA, we just said it would fail this criterion, and then the "However..." comes in and now makes me wonder again if it does or doesn't fail? Or is this an argument along the same lines as accessible alternatives?

Reply:
> IMO, in the situation quoted here, the user would meet AA even if they did not provide the additional content in a media alternative at A. They have fulfilled the wording of the AA requirement which really does seem to support the read that once the available space is used up, it's been met.

this is the understanding for the AAA one, so I'm confused why you're making the argument here about the AA?

Reply:
in short...I'm getting a bit lost and confused about your proposed rewrites. I'd say it'd be better, instead of doing the suggested change things here over multiple lines in the github interface, to maybe take a step back, and write out in full in a single comment what you're proposing here, what rationale you're trying to introduce, and if you want to mention it in the A or the AAA or both.

Reply:
> We're in the understanding for AAA

@patrickhlauke 
Okay, this may be my confusion. I though these changes were to address the question of whether a video with zero gaps in the dialogue can pass AA.
The title of the PR says it's for 1.2.3 and 1.2.5, and I was viewing it from the perspective of A and AA.
I agree this is messy.
Does it make sense for us to shove it into a document and work it there, and then only make the PR after we're relatively aligned on how to address?

Reply:
Hey, I just had a look and this is not for AAA, so I don't understand all your comments. But I still agree that trying to make significant changes is inelegant using the line-by-line Suggestions mechanism, and I think it would be easier in another medium.

Reply:
ah sorry, now it's me who got confused - could have sworn this was at AAA but I see it's AA. let me just take a step back and work out what the heck is going on then...